src/constants.js

```diff
@@ -1,14 +1,28 @@
 import React from 'react';
 import { Button } from '@patternfly/react-core';
+import { PageHeaderTitle } from '@redhat-cloud-services/frontend-components/PageHeader';
 import { dispatchNotification } from './Utilities/Dispatcher';
 import { getTimeDiff, renderRunDateTime } from './Utilities/helpers';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+import TasksPopover from './PresentationalComponents/TasksPopover/TasksPopover';
+import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
 
 export const TASKS_API_ROOT = '/api/tasks/v1';
 export const EXECUTED_TASKS_ROOT = '/executed_tasks';
+const ACCESS_REDHAT_DOT_COM =
+  'https://access.redhat.com/documentation/en-us/red_hat_insights/';
+const YEAR = '2022/html/';
```

Comment:
Wondering why the year is hard coded? Can we make it dynamic using [Date()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) class?